src/FSharp.Azure.Blob/Blob.fs

```diff
@@ -85,13 +86,33 @@ module Blob =
             }
         | _ -> failwith "This operation is valid only for Delete" 
 
-    let exists blobName options =
+    let exists (blobName: string) (options: ConnectionOperation) =
         Exists {
             Connection = options
             BlobName = Some blobName
         }
 
-    let overwriteBlob overwriteBlob (blobOperation: BlobOperation) = 
+    let getProperties (blobName: string) (options: ConnectionOperation) =
+        GetProperties {
+            Connection = options
+            BlobName = Some blobName
+        }
+
+    let setProperties (blobName: string) (properties: Azure.Storage.Blobs.Models.BlobHttpHeaders) (options: ConnectionOperation) =
+        SetProperties {
+            Connection = options
+            BlobName = Some blobName
+            Properties = Some properties
+        }
+
+    let setMetadata (blobName: string) (metadata: Collections.Generic.IDictionary<string,string>) (options: ConnectionOperation) =
+        SetMetadata {
+            Connection = options
+            BlobName = Some blobName
+            Metadata = Some metadata
```

Comment:
Don't need option when setting metadata, doesn't make much sense for someone setting metadata to None